scripts/igmp_query_injector.py

```diff
@@ -352,6 +209,7 @@ def inject_to_vifs_on_bridges(args):
     for bridge in args.bridges:
         if not igmp_snooping_is_enabled_on_bridge(bridge):
             log.info("IGMP snooping is disabled on bridge %s, won't inject query" % bridge)
+            continue
         bridges.append(bridge)
     injector = BridgeInjector(args.max_resp_time, bridges)
```

Comment:
I think that a separate BridgeInjector class adds unnecessary complexity. Can't you just calculate the list of vifs and then pass it to the VifInjector?

Reply:
Deleted this class and get vifs from bridges and inject IGMP by `IGMPQueryInjector`